docs/deployment/feature_gates.md

```diff
@@ -18,25 +18,26 @@ The following tables are a summary of the feature gates that you can set on diff
-| HVPAForShootedSeed                         | `false` | `Alpha` | `0.32` |        |
-| APIServerSNI                               | `false` | `Alpha` | `1.7`  | `1.18` |
-| APIServerSNI                               | `true`  | `Beta`  | `1.19` |        |
-| APIServerSNI (deprecated)                  | `true`  | `Beta`  | `1.48` |        |
-| HAControlPlanes                            | `false` | `Alpha` | `1.49` | `1.70` |
-| HAControlPlanes                            | `true`  | `Beta`  | `1.71` | `1.72` |
-| HAControlPlanes                            | `true`  | `GA`    | `1.73` |        |
-| DefaultSeccompProfile                      | `false` | `Alpha` | `1.54` |        |
-| CoreDNSQueryRewriting                      | `false` | `Alpha` | `1.55` |        |
-| IPv6SingleStack                            | `false` | `Alpha` | `1.63` |        |
-| MutableShootSpecNetworkingNodes            | `false` | `Alpha` | `1.64` |        |
-| FullNetworkPoliciesInRuntimeCluster        | `false` | `Alpha` | `1.66` | `1.70` |
-| FullNetworkPoliciesInRuntimeCluster        | `true`  | `Beta`  | `1.71` | `1.72` |
-| FullNetworkPoliciesInRuntimeCluster        | `true`  | `GA`    | `1.73` |        |
-| WorkerlessShoots                           | `false` | `Alpha` | `1.70` |        |
-| MachineControllerManagerDeployment         | `false` | `Alpha` | `1.73` |        |
+| Feature                             | Default | Stage   | Since  | Until  |
+|-------------------------------------|---------|---------|--------|--------|
+| HVPA                                | `false` | `Alpha` | `0.31` |        |
+| HVPAForShootedSeed                  | `false` | `Alpha` | `0.32` |        |
+| APIServerSNI                        | `false` | `Alpha` | `1.7`  | `1.18` |
+| APIServerSNI                        | `true`  | `Beta`  | `1.19` |        |
+| APIServerSNI (deprecated)           | `true`  | `Beta`  | `1.48` |        |
+| HAControlPlanes                     | `false` | `Alpha` | `1.49` | `1.70` |
+| HAControlPlanes                     | `true`  | `Beta`  | `1.71` | `1.72` |
+| HAControlPlanes                     | `true`  | `GA`    | `1.73` |        |
+| DefaultSeccompProfile               | `false` | `Alpha` | `1.54` |        |
+| CoreDNSQueryRewriting               | `false` | `Alpha` | `1.55` |        |
+| IPv6SingleStack                     | `false` | `Alpha` | `1.63` |        |
+| MutableShootSpecNetworkingNodes     | `false` | `Alpha` | `1.64` |        |
+| FullNetworkPoliciesInRuntimeCluster | `false` | `Alpha` | `1.66` | `1.70` |
+| FullNetworkPoliciesInRuntimeCluster | `true`  | `Beta`  | `1.71` | `1.72` |
+| FullNetworkPoliciesInRuntimeCluster | `true`  | `GA`    | `1.73` |        |
+| WorkerlessShoots                    | `false` | `Alpha` | `1.70` |        |
+| MachineControllerManagerDeployment  | `false` | `Alpha` | `1.73` |        |
+| DisableScalingClassesForShoots      | `false` | `Alpha` | `1.73` |        |
```

Comment:
Doesn't it make sense to enable the feature gate for in local setup for provider-local?

Reply:
Sure, we can enable this for the local provider – I also did this during testing. Did you see it make a difference for the actual resource usage on your machine? I guess in the local setup people have 1-2 control-planes and this is about requests only, not sure how this is handled in the local provider case.